### PR TITLE
Homework nr 8

### PR DESCRIPTION
**Pull request by Atem89**

### CODE REVIEW

File: src/main/java/homework_nr_8/NoDuplicate.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+public class NoDuplicate {
+    public List<String> noDuplicate(List<String> list){
+        List<String> duplicates = new ArrayList<>();
+        Set<String> set = new HashSet<>();
+
+        for (String i : list) {
```
Comment (gh3rtzz) [inline src/main/java/homework_nr_8/NoDuplicate.java:13]: This works but makes no sense. You could've used List here and get the same result. Try to: Set<String> set = new HashSet<>(duplicates);

**Review by gh3rtzz — CHANGES_REQUESTED** [review]
comment

File: src/main/java/homework_nr_8/NoDuplicate.java (hunk 1 above)
Comment (Atem89) [reply src/main/java/homework_nr_8/NoDuplicate.java:13]: Thank you.

Comment (gh3rtzz) [reply src/main/java/homework_nr_8/NoDuplicate.java:13]: now remove the for loop below, you don't need it anymore

Comment (Atem89) [reply src/main/java/homework_nr_8/NoDuplicate.java]: Thank you.

**Review by gh3rtzz — APPROVED** [review]
approved

**Pull request merged**